src/main/java/mate/academy/spring/dao/impl/UserDaoImpl.java

```diff
@@ -15,6 +15,16 @@ public UserDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
+    @Override
+    public Optional<User> getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            Query<User> query = session.createQuery("FROM User u "
+                    + "WHERE u.id = :id", User.class);
+            query.setParameter("id", id);
+            return query.uniqueResultOptional();
+        }
+    }
```

Comment:
```suggestion
    @Override
    public Optional<User> getById(Long id) {
        try (Session session = sessionFactory.openSession()) {
            return Optional.ofNullable(session.get(User.class, id));
        }
    }
```

We can not to complicate get by id methods in DAO